packages/upset/src/provenance/index.ts

```diff
@@ -60,7 +60,8 @@ const sortByAction = registry.register(
   'sort-by',
   (state, { sort, sortByOrder }) => {
     state.sortBy = sort;
-    state.sortByOrder = sortByOrder;
+    // should only be 'None' if sortBy is a Set
+    state.sortByOrder = sortByOrder || 'None';
```

Comment:
I think using `null` instead of `'none'` here is slightly more type-safe and makes sense semantically.

Reply:
Or does `state.sortByOrder` need to be a string?

Reply:
You're right that `state.sortByOrder` is a string. Using 'None' to me makes sense as there is no order for  sorting by set.